WordPress/Classes/Networking/MediaServiceRemoteXMLRPC.m

```diff
@@ -166,6 +166,26 @@ - (void)updateMedia:(RemoteMedia *)media
     }
 }
 
+- (void)deleteMedia:(RemoteMedia *)media
+            success:(void (^)())success
+            failure:(void (^)(NSError *))failure
+{
+    NSParameterAssert([media.mediaID longLongValue] > 0);
+
+    NSArray *parameters = [self XMLRPCArgumentsWithExtra:media.mediaID];
+    [self.api callMethod:@"wp.deleteFile"
```

Comment:
It's actually just an alias for `wp.deletePost`, which returns `true` if successful. I guess we should check that value.